PayButton/paybutton-server
# [#665] feat: resync when coming back

Related to #665


Depends on
---
- [x] #680




Description
---
Fixes issue where server being down for some time would make it miss all transactions that happened in the meanwhile.


Test plan
---
1. Have a button set up with one of your addresses,
2. Stop the `paybutton-dev` container (don't touch the other containers),
3. Send your address some XEC,
4. Start the `paybutton-dev` container again.


Executing the steps above on this branch should show the transaction made when the server was down, unlike in other branches.
